recipes/simdjson/0.2.1/conanfile.py

```diff
@@ -73,7 +74,7 @@ def _configure_cmake(self):
         cmake.definitions['SIMDJSON_DISABLE_AVX'] = not self.options.avx
         cmake.definitions['SIMDJSON_SANITIZE'] = False
         cmake.definitions['ENABLE_FUZZING'] = False
-        cmake.configure()
+        cmake.configure(source_folder=self._source_subfolder)
```

Comment:
This bypasses the exported cmake script of this recipe, which is unwanted.

Reply:
Ok.